app/services/get_system_metrics.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require 'sidekiq/api'
+
+class SystemMetricsService
+  def initialize
+    @queues = YAML.load_file('config/sidekiq.yml')[:queues]
+    fetch_sidekiq_stats
+  end
+
+  def fetch_sidekiq_stats
+    stats = Sidekiq::Stats.new
+
+    @processed_jobs_stat = stats.processed
+    @failed_jobs_stat = stats.failed # Failed Job Retention
+    @enqueued_jobs_stat = stats.enqueued
+    @scheduled_jobs_stat = stats.scheduled_size
+
+    # also allow stats from selected dates to be viewed
+    # using Stats::History
+  end
+
+  # def fetch_system_health_metrics
```

Comment:
My personal preference is to avoid leaving this kind of thing in the code, if it's not going to be part of a minimal initial implementation that we can merge and deploy. It think a first version that includes only sidekiq stats will already be useful, so we should probably plan on rolling that out before implementing any additional not-yet-defined metrics.